session/session_test.go

```diff
@@ -695,40 +695,44 @@ func Test_ProposeCh_CloseSession(t *testing.T) {
 		require.NoError(t, err)
 		assert.Len(t, persistedChs, 1)
 	})
-	t.Run("no_force_openChs", func(t *testing.T) {
-		ch, _ := newMockPCh(t, validOpeningBalInfo)
-		ch.On("Phase").Return(pchannel.Acting)
-		session := newSessionWCh(t, peerIDs, validOpeningBalInfo, ch)
 
-		_, err := session.Close(false)
-		require.Error(t, err)
-		t.Log(err)
-	})
 	t.Run("no_force_openChs", func(t *testing.T) {
 		ch, _ := newMockPCh(t, validOpeningBalInfo)
 		ch.On("Phase").Return(pchannel.Acting)
-		session := newSessionWCh(t, peerIDs, validOpeningBalInfo, ch)
+		newSessionWCh := newSessionWCh(t, peerIDs, validOpeningBalInfo, ch)
 
-		_, err := session.Close(false)
+		chsInfo, err := newSessionWCh.Close(false)
 		require.Error(t, err)
-		t.Log(err)
+
+		wantMessage := session.ErrOpenCh.Error()
```

Comment:
Can you rebase this PR on top of #150 ? There, I have added some assert helper functions (to the end of `session/session_test.go` file). Using those helper functions, will make these assert statements to one liners that are easy to read.